Examples/Framework/include/ActsExamples/EventData/Measurement.hpp

```diff
@@ -282,19 +290,49 @@ class MeasurementProxyBase {
     return self().subspaceHelper().expandMatrix(self().covariance());
   }
 
+  /// @brief Construct the measurement from a sourcelink, subspace vector,
+  /// parameters, and covariance.
+  ///
+  template <typename Subspace, typename ParameterDerived,
+            typename CovarianceDerived>
+  void construct(const Acts::SourceLink& source_link, Subspace&& subspace,
+                 const Eigen::DenseBase<ParameterDerived>& parameters,
+                 const Eigen::DenseBase<CovarianceDerived>& covariance)
+    requires(!ReadOnly) && requires {
+      { this->self().setSubspaceIndices(std::forward<Subspace>(subspace)) };
+      { this->self().parameters() = parameters };
+      { this->self().covariance() = covariance };
+    }
+  {
+    setSourceLink(source_link);
+    self().setSubspaceIndices(std::forward<Subspace>(subspace));
+    self().parameters() = parameters;
+    self().covariance() = covariance;
+  }
```

Comment:
I wonder if `construct` is a good name for the function as it does not really construct something but rather fill all the information? Maybe `fill` makes more sense? Again not a real issue

Reply:
Good idea, I like "fill".

Reply:
Done.